chibuezeayogu/React-Redux
# #161155240 Impletment course pagination

**What does the PR do?**
- This PR Implements  `Course Pagination`

**Description of Task to be completed?**
- paginate view course page
- update test
 
**How should this be manually tested?**
- Clone the repository [here](https://github.com/chibuezeayogu/React-Redux.git)
-  cd into `React-Redux` and checkout to this branch `ft-implement-pagination-161155240`
- run the command `yarn dev` to start the application locally.
- run the command `yarn test` to ensure all tests are passing.

**What are the relevant pivotal tracker stories?**
- [#161155240](https://www.pivotaltracker.com/story/show/161155240)

**Screenshots**
- N/A
